instaloader/structures.py

```diff
@@ -1862,7 +1862,7 @@ def name(self):
         return self._node["name"].lower()
 
     def _query(self, params):
-        json_response = self._context.get_json("explore/tags/{0}/".format(self.name), params)
+        json_response = self._context.get_json("api/v1/tags/web_info/?tag_name={0}".format(self.name), params)
```

Comment:
Nice, thanks for figuring this out and proposing this change!

This query type is already implemented as get_iphone_json. Thus, the following will do the request, making the proposed change in instaloadercontext.py unnecessary.

```suggestion
        json_response = self._context.get_iphone_json(
            "api/v1/tags/web_info/", {**params, "tag_name": self.name}
        )
```

Also, since this does no longer work when not logged-in, it would be good to annotate Instaloader.download_hashtag() with `@_requires_login` and to mention that in the --help output and in the docs (`basic-usage.rst`).